atomkraft/test/trace.py

```diff
@@ -55,4 +56,29 @@ def test_trace(trace: PathLike, reactor: PathLike, keypath: str):
             )
         )
     print(f"Executing {test_name} ...")
-    pytest.main(["-s", test_path])
+
+    report_dir = root / "reports" / test_name
+    report_dir.mkdir(parents=True)
+
+    logging_file = report_dir / "log.txt"
+
+    pytest_report_file = report_dir / "report.jsonl"
+
+    pytest_args = [
+        "--log-file-level=INFO",
+        f"--log-file={logging_file}",
+        f"--report-log={pytest_report_file}",
+    ]
+
+    if verbose:
+        pytest_args.append("-rP")
```

Comment:
I tried to generate a report for the transfer example. The jsonl file and the nodes directory are ok, but log.txt is empty. Maybe something to do with the arguments to pytest?

Reply:
`log.txt` will be empty if there was no logging out. Currently, none of our examples uses `logging` module.

Reply:
For now, let's leave it like this. Because this file is created by `pytest` directly.